skt/kerneltree.py

```diff
@@ -259,8 +259,18 @@ def __get_remote_url(self, remote):
             match = re.match('Fetch URL: (.*)', line)
             if match:
                 return match.group(1)
+        return None
 
     def __get_remote_name(self, uri):
+        """
+        Get the remote name of a specific URI.
+
+        Args:
+            uri: The URI of the repository's remote.
+
+        Returns:
+            Remote name's string, None if the uri doesn't exist.
```

Comment:
The wording is a bit awkward. The first part would sound better if it was something like `A string representing the name of the remote`, what do you think?

Reply:
What about `A string representing the remote name`?